frontend/src/lib/lemon-ui/LemonTextArea/LemonTextArea.tsx

```diff
@@ -105,43 +106,59 @@ export function LemonTextMarkdown({ value, onChange, ...editAreaProps }: LemonTe
 
     return (
         // Setting overflow: visible so that the LemonFileInput hover outline isn't clipped
-        <Tabs style={{ overflow: 'visible' }}>
-            <Tabs.TabPane tab="Write" key="write-card" destroyInactiveTabPane={true}>
-                <div ref={dropRef} className={clsx('LemonTextMarkdown flex flex-col space-y-1 rounded')}>
-                    <LemonTextArea ref={textAreaRef} {...editAreaProps} autoFocus value={value} onChange={onChange} />
-                    <div className="text-muted inline-flex items-center space-x-1">
-                        <IconMarkdown className={'text-2xl'} />
-                        <span>Markdown formatting support</span>
-                    </div>
-                    {objectStorageAvailable ? (
-                        <LemonFileInput
-                            accept={'image/*'}
-                            multiple={false}
-                            alternativeDropTargetRef={dropRef}
-                            onChange={setFilesToUpload}
-                            loading={uploading}
-                            value={filesToUpload}
-                        />
-                    ) : (
-                        <div className="text-muted inline-flex items-center space-x-1">
-                            <Tooltip title={'Enable object storage to add images by dragging and dropping.'}>
-                                <IconTools className={'text-xl mr-1'} />
-                            </Tooltip>
-                            <span>
-                                Add external images using{' '}
-                                <Link to={'https://www.markdownguide.org/basic-syntax/#images-1'}>
-                                    {' '}
-                                    Markdown image links
-                                </Link>
-                                .
-                            </span>
+        <LemonTabs
+            activeKey={isPreviewShown ? 'preview' : 'write'}
+            onChange={(key) => setIsPreviewShown(key === 'preview')}
+            tabs={[
+                {
+                    key: 'write',
+                    label: 'Write',
+                    content: (
+                        <div ref={dropRef} className={clsx('LemonTextMarkdown flex flex-col space-y-1 rounded')}>
```

Comment:
nit: clsx isn't really necessary here (just something i noticed, can go in either way ;-))